### PR TITLE
open-cluster-management/backlog#15656 add imp clst override

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by zhiweiyin318 — COMMENTED** [review]
looks good to me. thanks @cadawson

File: clusters/imp_clust_custom_image_override.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
+[#imp-clust-custom-image-override]
+= Importing a cluster with a custom ManagedClusterImageRegistry CRD
+
+There might be times when you need to override the registry images on the managed clusters that you are importing. You can do this by creating a `ManagedClusterImageRegistry` custom resource definition (CRD). 
+
+The `ManagedClusterImageRegistry` CRD is a namespace-scoped resource. link: deploy/foundation/hub/resources/crds/imageregistry.open-cluster-management.io_managedclusterimageregistries.yaml[Link didn't work. NEED A NEW LINK].
```
Comment (zhiweiyin318) [inline clusters/imp_clust_custom_image_override.adoc]: I don't think we need the link because it is not a formal path, may be changed in the future.

Comment (cadawson) [reply clusters/imp_clust_custom_image_override.adoc]: Thank you! I will remove the link.

**Comment by cadawson** [comment]
Link removed. Merging.

**Pull request merged**